crates/bevy_dev_tools/src/fps_overlay.rs

```diff
@@ -1,17 +1,28 @@
 //! Module containing logic for FPS overlay.
 
 use bevy_app::{Plugin, Startup, Update};
-use bevy_asset::Handle;
+use bevy_asset::{load_internal_asset, Asset, Assets, Handle};
 use bevy_color::Color;
 use bevy_diagnostic::{DiagnosticsStore, FrameTimeDiagnosticsPlugin};
 use bevy_ecs::{
     component::Component,
     query::With,
     schedule::{common_conditions::resource_changed, IntoSystemConfigs},
-    system::{Commands, Query, Res, Resource},
+    system::{Commands, Query, Res, ResMut, Resource},
 };
+use bevy_hierarchy::BuildChildren;
+use bevy_reflect::TypePath;
+use bevy_render::render_resource::{AsBindGroup, Shader, ShaderRef, ShaderType};
 use bevy_text::{Font, Text, TextSection, TextStyle};
-use bevy_ui::node_bundles::TextBundle;
+use bevy_ui::{
+    node_bundles::{MaterialNodeBundle, NodeBundle, TextBundle},
+    Style, UiMaterial, UiMaterialPlugin, Val,
+};
+use bevy_utils::default;
+
+const FRAME_TIME_GRAPH_SHADER_HANDLE: Handle<Shader> = Handle::weak_from_u128(2325577683959808);
+const FRAME_TIME_GRAPH_WIDTH_SCALE: f32 = 6.0;
+const FRAME_TIME_GRAPH_HEIGHT_SCALE: f32 = 2.0;
```

Comment:
should we make this customizable, instead of only scaling with `font_size`?

Reply:
Kinda, but I picked those values because they end up with a graph that is a similar size to the text and will scale automatically with it. I'd rather take care of this when working on a more complete DiagnosticOverlay since we'll need a better solution for scaling each element at that point.